tests/end_to_end/remoteTestDataVariants/variants/remoteTest/urls.json

```diff
@@ -0,0 +1,5 @@
+{
+    "urls": [
+        "http://0.0.0.0:8002/example_2.vcf.gz"
```

Comment:
@jeromekelleher do you have a recommendation for which vcf directory to serve for testing on the remote server here?

Serving a lot of stuff remotely gives me pysam errors, and example_2 doesn't seem to have a lot of data in it.  Here's a session which tries to open the same file remotely vs locally:
```
>>> a = 'http://0.0.0.0:8002/example_3.vcf.gz'
>>> b = pysam.VariantFile(a)
[E::hts_open] fail to open file 'http://0.0.0.0:8002/example_3.vcf.gz'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pysam/cbcf.pyx", line 2111, in pysam.cbcf.VariantFile.__init__ (pysam/cbcf.c:28920)
  File "pysam/cbcf.pyx", line 2254, in pysam.cbcf.VariantFile.open (pysam/cbcf.c:30878)
  File "pysam/cbcf.pyx", line 2305, in pysam.cbcf.VariantFile.open (pysam/cbcf.c:31539)
IndexError: tuple index out of range
>>> a = './tests/data/variants/example_3/example_3.vcf.gz'
>>> b = pysam.VariantFile(a)
>>>
```